source/_components/device_tracker.unifi.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Ubiquiti Unifi WAP"
+description: "Instructions how to use a Unifi WAP controller as a device tracker module."
+date: 2016-02-19 20:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ubiquiti.png
+ha_category: Presence Detection
```

Comment:
Should this be "Local Polling"?

https://github.com/balloob/home-assistant.io/blob/master/source/_components/light.hue.markdown

I guess I missed the start of the tagging effort, sorry about that. Excellent idea though.